homework/Vasili_Zasinets/Shape/src/main/java/com/tutrit/java/quickstart/shape/AbstractShape.java

```diff
@@ -0,0 +1,23 @@
+package com.tutrit.java.quickstart.shape;
+
+public abstract class AbstractShape implements Shape{
+    protected String name;
+
+    @Override
+    public double getArea() throws ShapeLengthException {
+        return 0;
```

Comment:
I'm confused. Why just 0? Need to be clarified: either exception or log message should be fine.

Reply:
fixed